CMakeLists.txt

```diff
@@ -216,11 +220,11 @@ macro(xeus_create_target target_name linkage output_name)
     # =================
 
     target_compile_features(${target_name} PRIVATE cxx_std_14)
+    xeus_target_add_compile_warnings(${target_name})
 
     if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR
         CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR
         CMAKE_CXX_COMPILER_ID MATCHES "Intel")
-        target_compile_options(${target_name} PUBLIC -Wunused-parameter -Wextra -Wreorder)
```

Comment:
I think we're missing `extra` and `reorder` in the cmake warning file.

Reply:
`extra` already there. I could not find a `reorder` warning on GCC doc.

Reply:
Althoug there is no a detailed documentation about it, we can find it in the list of warnings enabled by `-Wall`: https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html